crates/analyzer/tests/errors.rs

```diff
@@ -340,10 +340,10 @@ test_ingot! { mainless_ingot }
 test_ingot! { bad_visibility }
 test_ingot! { missing_dep }
 test_ingot! { misconfigured_dep }
-#[cfg(not(target_arch = "wasm32"))]
-test_ingot! { dep_unavailable_rev }
-#[cfg(not(target_arch = "wasm32"))]
-test_ingot! { dep_unavailable_source }
+// #[cfg(not(target_arch = "wasm32"))]
+// test_ingot! { dep_unavailable_rev }
+// #[cfg(not(target_arch = "wasm32"))]
+// test_ingot! { dep_unavailable_source }
```

Comment:
it looks like the git2 crate we're using for dep resolution isn't returning errors for non-existent repositories